tabuckner/broskichat
# Integrate socketio fe

### Project Changed


**- What I Did**
Integrated socket.io for Front End

**- How I Did It**
- Added a proof of concept for FE usage. NOT CLEAN AND SHOULD BE IMPROVED ON
- Made small tweaks to BE integration
- Tested
**- How to Test/Verify It**
- Launch API `yarn start:watch`
- Launch Client `yarn start`
- Navigate to chat route
- Send a test message

You should Receive an echoed message from the server. 
Test again til you're done goofin around
![image](https://user-images.githubusercontent.com/32392635/44242448-5ef04500-a18f-11e8-9c8e-a1c8ea00ce4f.png)


**- An Image to Make Us Giggle (not mandatory but encouraged)**
